tests/bearlib/abstractions/external_bear_wrap_testfiles/test_external_bear.py

```diff
@@ -1,32 +1,46 @@
+import os
 import sys
 import json
 
 if __name__ == "__main__":
 
+    sys.path.append(os.path.join("..", "..", "..", ".."))
+
+    from coalib.results.Result import Result
+    from coalib.results.SourceRange import SourceRange
+    from coalib.output.JSONEncoder import create_json_encoder
+    from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
```

Comment:
indeed it is put, but in my case I use ` if __name__ == "__main__":` to avoid this getting ran as a test. If it is ran as a test it doesn't need to do the `sys.path.append` or the other imports